cernopendata/modules/fixtures/data/records/jade-computing-notes.json

```diff
@@ -98,13 +98,13 @@
     "accelerator": "DESY-PETRA",
     "authors": [
       {
-        "name": "Hagermann"
+        "name": "Hagermann, J."
```

Comment:
The bibliographic standard is to leave the initial without the dots, i.e. "Hagermann, J" format.  This will better compare to what CDS, INSPIRE etc repositories are doing, and will also play more nicely with facets here when the first name is fully known. 

Can you please amend the names to leave out trailing dots everywhere?